PokeDemo/Extensions/UITableViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UITableViewCell.swift
```

Comment:
Rename UITableViewCell+ReuseIdentifying

Reply:
Covered in bf657192bcbf9c8546d097f1262e9261d6bac8d2